app/helpers/passengers_helper.rb

```diff
@@ -6,15 +6,16 @@ def passengers_table_row_class(passenger)
     return if passenger.permanent?
     note = passenger.doctors_note
     if note.present?
-      if note.will_expire_within_warning_period?
+      if note.override_expiration?
+        'overridden'
+      elsif note.will_expire_within_warning_period?
         'will_expire_soon'
       elsif note.expired_within_grace_period?
         'expired_within_grace_period'
       elsif note.expired?
```

Comment:
So if the `note` has expired, that means the `passenger` is `inactive`? And an `inactive passenger` will always have an `expired note`? I'm asking because both return `'inactive'`.

Reply:
Correct. Activeness takes precedence, is why you see it here twice. An inactive passenger is inactive regardless of anything to do with their note. An active passenger is shown as inactive if their note is expired (beyond the grace period).

Reply:
Alright, awesome.